farm/inference_rest_api.py

```diff
@@ -84,7 +84,7 @@ def post(self, model_id):
         dicts = request.get_json().get("input", None)
         if not dicts:
             return {}
-        results = model.inference_from_dicts(dicts=dicts, rest_api_schema=True)
+        results = model.inference_from_dicts(dicts=dicts)
         return results[0]
```

Comment:
results are not a list of QApreds, right_? Is this the right way to return those?

Reply:
rest_api_schema has been replaced by return_json. By default, return_json is True so we will get a dict output here instead of QAPreds

Reply:
Ah ok thanks for clarification.